addons/website/static/src/snippets/s_timeline/options.js

```diff
@@ -27,8 +27,9 @@ options.registry.Timeline = options.Class.extend({
      */
     timelineCard(previewMode, widgetValue, params) {
         const timelineRowEl = this.$target[0].closest(".s_timeline_row");
+        const timelineCardsEls = timelineRowEl.querySelectorAll('.s_timeline_card');
```

Comment:
Nitpick: rename it as `timelineCardEls` (without the "s" at the end of "Cards") :slightly_smiling_face: